### PR TITLE
LPS-88751

### PR DESCRIPTION
**Pull request by jonmak08**
Hey @drewbrokke,

Attached is an update for http://issues.liferay.com/browse/LPS-88751.

Can you look this over to make sure my changes make sense and do not need to be further updated?  If it is good, you can forward this to @brianchandotcom for final review.

Please let me know if you have any questions. Thanks!

### CODE REVIEW

File: modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/js/UserNameFields.es.js [hunk 1]
```diff
@@ -1,120 +1,141 @@
...
-					resolve(url);
-				}
-			);
-		});
+		return new Promise(
+			(resolve) => {
+				AUI().use(
+					'liferay-portlet-url',
+					A => {
+						const url = Liferay.PortletURL.createURL(this.baseURL);
```
Comment (drewbrokke) [inline modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/js/UserNameFields.es.js]: Should we remove the parens from the `resolve` argument above also?

Comment (jonmak08) [reply modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/js/UserNameFields.es.js]: Whoops, missed that.  Yes, go ahead and remove that too.  Thanks!

**Comment by drewbrokke** [comment]
Just started reviewing :)

**Comment by drewbrokke** [comment]
PR sent to https://github.com/brianchandotcom/liferay-portal/pull/66821

**Pull request closed without merging**